Library/Homebrew/download_strategy.rb

```diff
@@ -373,8 +373,9 @@ class CurlDownloadStrategy < AbstractFileDownloadStrategy
 
   attr_reader :mirrors
 
-  def initialize(url, name, version, **meta)
+  def initialize(url, name, version, try_partial: true, **meta)
```

Comment:
Do we ever set this to false? If not, could just remove this argument.

Reply:
> Do we ever set this to false? If not, could just remove this argument.

`CurlDownloadStrategy` uses the previous `try_partial` default (`true`) everywhere, so no, not at the moment. I was mostly providing it as an option for the future but I'm fine leaving it off until it's needed.

If we leave it off, are you still fine with the `@try_partial` instance variable setup? It's arguably nicer than hardcoding the value in all the `#curl_download` calls.

Reply:
Yup, fine with me to keep the instance variable!